MaterialSkin/Controls/MaterialExpansionPanel.cs

```diff
@@ -0,0 +1,607 @@
+
+            g.Clear(Parent.BackColor);
+
+            // card rectangle path
+            RectangleF expansionPanelRectF = new RectangleF(ClientRectangle.Location, ClientRectangle.Size);
+            expansionPanelRectF.X -= 0.5f;
+            expansionPanelRectF.Y -= 0.5f;
+            GraphicsPath expansionPanelPath = DrawHelper.CreateRoundRect(expansionPanelRectF, 2);
+
+            // button shadow (blend with form shadow)
+            DrawHelper.DrawSquareShadow(g, ClientRectangle);
+
+            // Draw expansion panel
+            // Disabled
+            if (!Enabled)
+            {
+                using (SolidBrush disabledBrush = new SolidBrush(DrawHelper.BlendColor(Parent.BackColor, SkinManager.BackgroundDisabledColor, SkinManager.BackgroundDisabledColor.A)))
+                {
+                    g.FillPath(disabledBrush, expansionPanelPath);
+                }
+            }
+            // Mormal
+            else
+            {
+                if ((_buttonState == ButtonState.HeaderOver | _buttonState == ButtonState.ColapseExpandOver) && _collapse)
+                {
+                    RectangleF expansionPanelBorderRectF = new RectangleF(ClientRectangle.X + 1, ClientRectangle.Y + 1, ClientRectangle.Width - 2, ClientRectangle.Height - 2);
+                    expansionPanelBorderRectF.X -= 0.5f;
+                    expansionPanelBorderRectF.Y -= 0.5f;
+                    GraphicsPath expansionPanelBoarderPath = DrawHelper.CreateRoundRect(expansionPanelBorderRectF, 2);
+																					   
+                    g.FillPath(SkinManager.ExpansionPanelFocusBrush, expansionPanelBoarderPath);
+                }
+                else
+                {
+                    using (SolidBrush normalBrush = new SolidBrush(SkinManager.BackgroundColor))
+                    {
+                        g.FillPath(normalBrush, expansionPanelPath);
+                    }
+                }
+            }
+
+            // Calc text Rect
+            Rectangle headerRect = new Rectangle(
+                _leftrightPadding,
+                (_headerHeight - _textHeaderHeight) / 2,
+                TextRenderer.MeasureText(_titleHeader, Font).Width + _expansionPanelDefaultPadding,
+                _textHeaderHeight);
+
+            //Draw  headers
+            using (NativeTextRenderer NativeText = new NativeTextRenderer(g))
+            {
+                // Draw header text
+                NativeText.DrawTransparentText(
+                    _titleHeader,
+                    SkinManager.getLogFontByType(MaterialSkinManager.fontType.Body1),
+                    Enabled ? SkinManager.TextHighEmphasisColor : SkinManager.TextDisabledOrHintColor,
+                    headerRect.Location,
+                    headerRect.Size,
+                    NativeTextRenderer.TextAlignFlags.Left | NativeTextRenderer.TextAlignFlags.Middle);
+            }
+
+            if (String.IsNullOrEmpty(_descriptionHeader) == false)
```

Comment:
no need to compare a boolean operation, just use `!String.IsNullOrEmpty(_descriptionHeader)`